Airports/EGVN/Fixes.txt

```diff
@@ -0,0 +1,12 @@
+D078B N051.45.20.722 W001.33.04.331
+D282J N051.46.46.780 W001.52.00.919
+D255P N051.40.18.001 W002.01.05.999
+D252C N051.44.03.120 W001.40.00.001
+D076I N051.47.15.781 W001.22.14.138
+D254H N051.42.34.139 W001.48.31.010
+D271J N051.44.49.039 W001.52.17.900
+D036J N051.53.05.129 W001.25.59.798
+D358J N051.54.51.631 W001.37.11.219
+D243J N051.40.06.920 W001.50.19.702
+D197J N051.35.16.591 W001.40.33.218
+D225J N051.37.39.529 W001.47.17.440
```

Comment:
These need to be unique, D078B is in KK SIDs, D254H is in CC SIDs

Reply:
Changed and added note about the conflict.